openfisca_core/columns.py

```diff
@@ -19,6 +19,11 @@ def N_(message):
 
 year_or_month_or_day_re = re.compile(ur'(18|19|20)\d{2}(-(0?[1-9]|1[0-2])(-([0-2]?\d|3[0-1]))?)?$')
 
+# Possible values for period_behavior
+MONTH = u'month'
+YEAR = u'year'
+PERMANENT = u'permanent'
```

Comment:
How does an enum works ?

What happen if I do `print(QUIFAM['chef'])` ? Does it just print a number ?